spec/classes/nftables_spec.rb

```diff
@@ -71,18 +80,34 @@
         )
       }
 
-      it {
-        is_expected.to contain_systemd__dropin_file('puppet_nft.conf').with(
-          content: %r{^ExecReload=/sbin/nft -I /etc/nftables/puppet -f /etc/sysconfig/nftables.conf$},
-        )
-      }
+      if os_facts[:os]['family']  == 'Debian'
```

Comment:
once access as through a symbol and once as string? Is this really needed like that?

Reply:
If `:os` is a string `'os'`, the code fails with ```
Failure/Error: if os_facts['os']['family']  == 'Debian'

NoMethodError:
  undefined method `[]' for nil:NilClass
``` 

However, it seems both than be symbols.

Reply:
I've updated it to use symbols for both parts of the lookup